### PR TITLE
Improved the cross-schema behavior

### PR DESCRIPTION
**Pull request by yallie**
* Added `set search_path from current` clauses so that `pgunit` functions are guaranteed to see each other regardless of the current search_path set by the user. This allows installing the framework into a dedicated schema, event not present in the `search_path`.
* Added auto-detection of the `dblink` extension schema. Autonomous transactions will work even if dblink extension is installed to a non-standard location, not present in the `search_path`.
* Updated the `README.md` file to merge two Troubleshooting sections into one.

Now it works fine even if I can't control my `search_path`.
I installed the framework in a `pgunit` schema and qualify its calls, i.e.:
```sql
perform pgunit.test_assertNotNull('test should not be null', test);
```

### CODE REVIEW

File: PGUnit.sql [hunk 1]
```diff
@@ -192,26 +192,101 @@ $$ language sql;
 --
 create or replace function test_autonomous(p_statement VARCHAR) returns void as $$
 declare
-    l_error_text CHARACTER VARYING;
```
Comment (yallie) [inline PGUnit.sql:195]: Updated formatting here so that it's consistent with the rest of the framework.

**Review by adrianandrei-ca — APPROVED** [review]
Looks good to me.

**Comment by yallie** [comment]
Thanks!

**Comment by adrianandrei-ca** [comment]
Highly appreciate your contribution to the project. Thank you and I hope
you find this product useful.

**Comment by yallie** [comment]
Thank you for the great framework! It's easy to use and understand.

**Pull request merged**